mmedit/engine/optimizers/pggan_optimizer_constructor.py

```diff
@@ -18,8 +18,8 @@ class PGGANOptimWrapperConstructor:
     the scale is the index of the resolution scale.
 
     More detail about the resolution scale and naming rule please refers to
-    :class:~`mmgen.models.PGGANGenerator` and
-    :class:~`mmgen.models.PGGANDiscriminator`.
+    :class:~`mmedit.models.PGGANGenerator` and
```

Comment:
```suggestion
    :class:`~mmedit.models.editors.pggan.PGGANGenerator` and
```

 latest dev-1.x has resolved it